packages/client/hmi-client/src/services/models/funman-service.ts

```diff
@@ -179,14 +178,12 @@ export const renderFumanTrajectories = (
 	d3.select(element).selectAll('*').remove();
 	const svg = elemSelection.append('svg').attr('width', width).attr('height', height);
 
-	const points = trajs.filter((d: any) => d.boxId === boxId);
-
```

Comment:
This doesn't look right, the plots, they do not look like mathematical functions

If we are to remove the boxId-filter, then we need to "group by" the boxIds.

What you have here is that you aggregated all the boxes together and it can give weird results.

Reply:
Thanks for catching this!